projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3664,7 +3821,13 @@ public static function get_plugin_update_count() {
 			} else {
 				$response = array(
 					'code'    => 'updates-available',
-					'message' => esc_html( sprintf( _n( '%s plugin need updating.', '%s plugins need updating.', $count, 'jetpack' ), $count ) ),
+					'message' => esc_html(
+						sprintf(
+							/* Translators: placeholders are numbers. */
+							_n( '%s plugin need updating.', '%s plugins need updating.', $count, 'jetpack' ),
```

Comment:
Should we fix the grammar in the message while we're here?
```suggestion
							_n( '%s plugin needs updating.', '%s plugins need updating.', $count, 'jetpack' ),
```